build.sh

```diff
@@ -23,15 +23,19 @@ source "$REPOROOT/scripts/build/process-args.sh"
 # once that is fixed, we should remove the NOPACKAGE flag and do the full build either in
 # or out of docker.
 if [ ! -z "$BUILD_IN_DOCKER" ]; then
-    export BUILD_COMMAND="/opt/code/scripts/build/build.sh NOPACKAGE"
+    export BUILD_COMMAND=". /opt/code/scripts/build/process-args.sh $@ ; . /opt/code/scripts/build/build.sh"
```

Comment:
We need to propagate the args (like `debug`, `release`, `nocache`) into the docker container. Before the fix none of these options were passed into the docker container and the build uses the default options. As a result we were never building release config inside the docker container.